articles/defender-for-cloud/integration-defender-for-endpoint.md

```diff
@@ -326,6 +326,12 @@ If you enabled the integration, but still don't see the extension running on you
 ### What are the licensing requirements for Microsoft Defender for Endpoint?
 Defender for Endpoint is included at no extra cost with **Microsoft Defender for Servers**. Alternatively, it can be purchased separately for 50 machines or more.
 
+### Do I need to buy a separate antivirus solution to protect my machines?
+No. With MDE integration in Defender for Servers, you will also get antimalware protection on your machines.
```

Comment:
I changed the FAQ so it now talks about anti-malware instead of antivirus. Anti-malware is the more modern approach, covering not only viruses, but also Trojans, Worms, and other malicious software.